### ISSUE TITLE
TSC Meeting 2024-04-18

### ISSUE THREAD

**Issue by maxnbk** [AcademySoftwareFoundation/rez#1727]
Please respond to this issue with items you would like to see added to the agenda.

Items from attendees of TSC Meetings are prioritized above others, though ad-hoc reprioritization and discussion will occur.

Current agenda below:

* ASWF
  * TAC Project Update occurred, rez renewed another year.
* @isohedronpipeline topics:
  * https://github.com/AcademySoftwareFoundation/rez/pull/1706
  * https://github.com/AcademySoftwareFoundation/rez/pull/1709

**Comment by brycegbrazen** [comment]
Think we could circle back around to #1668 and see if we can come up with action items?

**Comment by brycegbrazen** [comment]
Also curious about #1475 and if we can get that one closed soon. I know that's a big one.

**Comment by loonghao** [comment]
@maxnbk Will there still be meeting notes and screen recordings? I found that there have been no meeting notes or screen recordings for a long time.

https://github.com/AcademySoftwareFoundation/rez/tree/main/ASWF/TSC/meeting-notes

Thank you very much for the information you recorded before, which allows us to better understand the detailed of TSC discussion.

**Comment by maxnbk** [comment]
@loonghao , Yes, we're a bit behind on meeting notes. It's a lot to write ;) We'll try to get the backlog processed sooner than later.
